kubectl-rook-ceph.sh

```diff
@@ -245,6 +249,20 @@ function run_rook_cr_status() {
   fi
 }
 
+function run_purge_osd() {
+  force_removal=false
+  if [ "$#" -eq 2 ] && [ "$2" = "--force" ]; then
```

Comment:
In the future when we have more complex commands it would be nice to allow arguments in a different order, but this is ok for now.

Reply:
Good point. Why aren't we using the arg parsing function here?

Reply:
IIUC, the `parse_flags` method is written, only accept flags starting before command eg `kuecttl rook-ceph --namesapce ...` and here we are passing `--force` in the end of the command.
But I do agree it is not the best code and also I found some minor bugs. I'm not planning to fix the minor bugs in the pr.  We can discuss about `parge_flags`

Reply:
The currently commented-out status subcommand has flags after a command. See `parse_status_flag`.